articles/openshift/oshift-how-use-portworx-shared-volumes.md

```diff
@@ -0,0 +1,238 @@
+---
+title: How to use Portworx shared volumes | UKCloud Ltd
+description: Explains how to use shared volumes in a Portworx integrated OpenShift cluster
+services: openshift
+author: Ben Bacon
+reviewer:
+lastreviewed: 16/10/2019
+
+toc_rootlink: How To
+toc_sub1: Use Portworx with OpenShift
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Use Portworx shared volumes
+toc_fullpath: How To/Use Portworx with OpenShift/oshift-how-use-portworx-shared-volumes.md
+toc_mdlink: oshift-how-use-portworx-shared-volumes.md
+---
+
+# How to use Portworx shared volumes
+
+## Overview
+
+Portworx is a cloud-native storage solution that is now available as an integration with our OpenShift offering. This article describes how to create shared volumes.
+
+### Intended audience
+
+This article assumes you have access to a Portworx integrated OpenShift 3.11 cluster and that you have cluster-admin rights. It also assumes familiarity with `oc`, the OpenShift command-line client. 
+
+If you're interested in a free 30 day trial of Portworx, raise a Service Request via the [My Calls](https://portal.skyscapecloud.com/support/ivanti) section of the UKCloud Portal.
+
+## Shared volumes
+
+Portworx enables persistent volumes claims to be mounted to multiple containers across multiple nodes enabling ease of pod-scaling.
```

Comment:
```suggestion
Portworx enables persistent volumes to be mounted to multiple containers across multiple nodes.
```
If you want to talk more specifically about claims here you can, but in general what we're mounting is the PV not the PVC. 
Also is it enabling ease of pod scaling? For me it's enabling volumes to be accessible across multiple pods - there are lots of complications that come from a shared filesystem potentially so I'm not sure its ease of pod-scaling so much.